spec.html

```diff
@@ -22815,16 +22843,23 @@ <h1>Exports</h1>
       <h2>Syntax</h2>
       <emu-grammar type="definition">
         ExportDeclaration :
-          `export` `*` FromClause `;`
-          `export` ExportClause FromClause `;`
-          `export` ExportClause `;`
+          `export` ExportFromClause FromClause `;`
```

Comment:
To me, "ExportFromClause" seems like the wrong name. The nonterminal 'FromClause' is so called because it's a clause that starts with the keyword "from". The nonterminal "ExportFromClause" seems like it should have a similar story, but doesn't.

Maybe "ReExports"? Or I suppose "ExportClause" is justified on the basis of symmetry with the Import productions. (In any case, renaming the *current* "ExportClause" to "NamedExports" makes sense to me.)

Alternatively, just inline the new production:
```
ExportDeclaration :
    `export` `*` FromClause `;`
    `export` `*` `as` IdentifierName FromClause `;`
    `export` NamedExports FromClause `;`
    ...
```

Reply:
Yes, I was thinking along the same lines. I would prefer the inlining solution, unless there's some other reason for the additional non-terminal?

Reply:
Inlining is getting me a very hard time at `Static Semantics: ExportEntries`